Assets/MirageXR/Tests/NewUI/ProfileView.cs

```diff
@@ -278,9 +282,9 @@ private void OnClickPrivacyPolicy()
 
     private void UpdatePrivacyPolicyButtonActive()
     {
-        var setActive = (DBManager.privacyPolicyDomain != string.Empty) ? true : false;
+        //var setActive = (DBManager.privacyPolicyDomain != string.Empty) ? true : false;
 
-        _btnPrivacyPolicy.gameObject.SetActive(setActive);
+        //_btnPrivacyPolicy.gameObject.SetActive(setActive);
```

Comment:
these changes should be here?

Reply:
not necessarily, it is discussed in the [PR_1347](https://github.com/WEKIT-ECS/MIRAGE-XR/pull/1346)